config/ignite-log4j2.xml

```diff
@@ -28,6 +28,12 @@
             <PatternLayout pattern="[%d{ISO8601}][%-5p][%t][%c{1}]%notEmpty{[%markerSimpleName]} %m%n"/>
         </Console>
 
+        <File name="CONSISTENCY" fileName="${sys:IGNITE_HOME}/work/log/consistency.log">
```

Comment:
Why not APPEND here?

Reply:
Should not be added to relular log, I think.

Reply:
append is true by default
https://logging.apache.org/log4j/2.x/manual/appenders.html